tests/workorder_json.py

```diff
@@ -0,0 +1,212 @@
+import json
```

Comment:
Generic comment change all function and variable names to snake case wherever it is necessary.

Reply:
Done.